### ISSUE TITLE
Please create a github pages site for the ip address. Us api devs need love too. 

### ISSUE THREAD

**Issue by baash05** [NathanaelA/internetprotests#3]

**Comment by NathanaelA** [comment]
I'm not opposed to it; I just don't understand how it helps/enhances anything...

You can grab the raw file directly from: https://github.com/NathanaelA/internetprotests/blob/master/ips/ips.txt 
 or 
http://internetprotests.com/ips/ips.txt

Does github pages offer something beyond that which I'm unaware of?

**Comment by baash05** [comment]
Yeah.. when you update the page the file can be read from. Actually, with
this direct link, we could parse it.
Thanks.

------------------------------------------------
You know that I write slowly. This is chiefly because I am never satisfied
until I have said as much as possible in a few words, and writing briefly
takes far more time than writing at length.
— Karl Friedrich Gauss
-----------------------------------------------
